docs/en/solutions/ArtifactPromotionRun_Approval_Notification_with_CorpWeChat.md

```
@@ -0,0 +1,491 @@
+```
+
+#### Required Parameter Customization
+
+Customize the following parameters for each user:
+
+| Parameter | Description | Example |
+|-----------|-------------|---------|
+| `metadata.name` | Unique resource name identifying the subscription | `admin-artifactpromotionrun-subscription` |
+| `metadata.annotations["katanomi.dev/owned.username"]` | Username of the notification recipient (must match `spec.subscriber.name`) | `admin` |
+| `spec.subscriber.name` | Kubernetes username of the subscriber | `admin` |
+| `spec.subscriber.info.id` | ACP user ID, you can retrive it by `kubectl get users | grep {subscriber.name}` | `21232f297a57a5a743894a0e4a801fc3` |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)